app/index.js

```diff
@@ -16,7 +16,7 @@ app.set('view engine', '.hbs');
 const port = 8000;
 
 const client = new smartcar.AuthClient({
-  testMode: true,
+  mode: "test",
```

Comment:
```suggestion
  mode: 'test',
```

consistency with rest of the file